app/models/form5655_submission.rb

```diff
@@ -10,6 +10,18 @@ class StaleUserError < StandardError; end
   has_kms_key
   has_encrypted :form_json, :metadata, key: :kms_key, **lockbox_options
 
+  scope :streamlined, -> { where("(public_metadata -> 'streamlined' ->> 'value')::boolean") }
```

Comment:
Would the duplicate code here and the debts_api/v0/form5655_submission.rb model benefit from a shared concern to dry up to the code?

Reply:
We are in the process of moving our codebase to the debts_api module. The duplication between debt_api and debt_management_center is temporary while we test for parody.